frontend/shared/src/components/benefit/tabs/Tabs.sc.ts

```diff
@@ -1,13 +1,17 @@
 import { TabList, TabPanel } from 'hds-react';
```

Comment:
the file path is > `frontend/shared/src/components/benefit/tabs/Tabs.sc.ts` Should the contents of the `benefit` folder be in `benefit-shared` instead?